### PR TITLE
Order processing exception from hasRefundingEnabled

### LINKED ISSUE #2705: [Marketplace] Order processing exception from hasRefundingEnabled
In [marketplace] branch, order processing produces browser error `Option in template helper: TypeError: Cannot read property 'support' of undefined`, which is an uncaught exception from the `hasRefundingEnabled` method.

To replicate:

Shippo + PayPal Express payment method.  Place order, attempt to being processing order in admin.

cart_completed_and_inbox__9_766_messages__123_unread_

## version

Node: 6.11.1
NPM: 5.3.0
Meteor Node: 4.8.4
Meteor NPM: 4.6.1
Reaction CLI: 0.11.0
Reaction: 1.5.0
Reaction branch: marketplace
Docker: 17.06.0-ce

### PR DESCRIPTION
**Pull request by joykare**
Resolves #2705 

### Behaviour
Order processing produces browser error Option in template helper: 
`TypeError: Cannot read property 'support' of undefined, which is an uncaught exception from the hasRefundingEnabled method.`

### Expected behaviour
Not to have error.

### How to test
- Place order using PayPal Express payment method
- Open console
- Inspect for errors as you process the order

### CODE REVIEW

**Comment by joykare** [comment]
**Status**
On WIP just coz tests keep failing on CI (not sure why, coz they are passing locally)

**Comment by zenweasel** [comment]
@aaronjudd We're just waiting for you on this?

File: imports/plugins/included/payments-paypal/register.js [hunk 1]
```diff
@@ -9,9 +9,9 @@ Reaction.registerPackage({
     express: {
       enabled: false,
       support: [
-        "authorize",
```
[the remark this answers is not in the document]
Comment (zenweasel) [reply imports/plugins/included/payments-paypal/register.js]: @aaronjudd I introduced this bug/error above by changing the `check` statement on that method. @foladipo pointed this out to me and changed it in https://github.com/reactioncommerce/reaction/pull/2738

I can also create a PR just to make that change if it's blocking this PR

[the remark this answers is not in the document]
Comment (spencern) [reply imports/plugins/included/payments-paypal/register.js]: Agreed that we should use consistent values when inserting anything into the database. We should codify this as - "Use camelCase when inserting text values into the database" probably.

In my shopify sync I've got some data that doesn't follow this pattern, for reason, so any comments there would be appreciated too.
https://github.com/reactioncommerce/reaction/pull/2808/files#diff-495a43ae56c5c6439438bd9c1a30eac3R101

**Pull request merged**